src/app/home/home-banner/home-banner.component.html

```diff
@@ -3,4 +3,14 @@
         <h2>{{bigText}}<span class="roboto">.</span></h2>
         <h4>{{subText}}<span class="roboto">.</span></h4>
     </div>
+    <ng-container *ngFor="let image of images">
+        <img
+        alt=""
+        class="ball"
+        [src]='getUrl(image)'
+        [class.bottom]="image.layer === 'bottom'"
+        [class.front]="image.layer === 'front'"
```

Comment:
If the class is either `bottom` or `front`, why not using `ngClass` to pass the `image.layer` content as the class?